docs/advanced/peering/offloading-in-depth.md

```diff
@@ -0,0 +1,251 @@
+  kubeconfigSecretRef:
+    name: kubeconfig-resourceslice-mypool
+  labels:
+    liqo.io/provider: kubeadm
+    liqo.io/remote-cluster-id: cool-firefly
+  resourceQuota:
+    hard:
+      cpu: "4"
+      ephemeral-storage: 20Gi
+      memory: 8Gi
+      pods: "110"
+  storageClasses:
+  - storageClassName: liqo
+status: {}
+```
+````
+
+`````
+
+This command will create a `VirtualNode` named `mynode` in the consumer cluster, and it will be associated with the `cool-firefly` cluster.
+
+```bash
+kubectl get virtualnodes.offloading.liqo.io -A      
+NAMESPACE                  NAME     CLUSTERID      CREATE NODE   AGE
+liqo-tenant-cool-firefly   mynode   cool-firefly   true          7s
+```
+
+A new `Node` will be available in the consumer cluster with the name `mynode` providing the resources specified in the `ResourceSlice`.
+
+```bash
+kubectl get node
+NAME                            STATUS   ROLES           AGE   VERSION
+cluster-1-control-plane-fsvkj   Ready    control-plane   52m   v1.27.4
+cluster-1-md-0-dzl4s            Ready    <none>          52m   v1.27.4
+mynode                          Ready    agent           22s   v1.27.4
+```
+
+```{warning}
+If you create multiple `VirtualNodes` using the same `ResourceSlice`, the resources will be shared among them.
+```
+
+### With Existing Secret
+
+If you have a `kubeconfig` secret in the consumer cluster, you can create a `VirtualNode` that will use the resources specified in the `kubeconfig` secret.
+
+`````{tab-set}
+
+````{tab-item} liqoctl
+```bash
+liqoctl create virtualnode --remote-cluster-id cool-firefly --kubeconfig-secret-name kubeconfig-resourceslice-mypool mynode
+```
+````
+
+````{tab-item} YAML
+```yaml
+apiVersion: offloading.liqo.io/v1beta1
+kind: VirtualNode
+metadata:
+  creationTimestamp: null
+  labels:
+    liqo.io/remote-cluster-id: cool-firefly
+  name: mynode
+  namespace: liqo-tenant-cool-firefly
+spec:
+  clusterID: cool-firefly
+  createNode: true
+  disableNetworkCheck: false
+  kubeconfigSecretRef:
+    name: kubeconfig-resourceslice-mypool
+  labels:
+    liqo.io/remote-cluster-id: cool-firefly
+  resourceQuota:
+    hard:
+      cpu: "2"
+      memory: 4Gi
+      pods: "110"
+```
+````
+
+`````
+
+The `kubeconfig` secret must be created in the consumer cluster in the same namespace where the `VirtualNode` will be created.
+The secret must contain the `kubeconfig` file of the provider cluster.
+
+```yaml
+apiVersion: v1
+data:
+  kubeconfig: <base64-encoded-kubeconfig>
+kind: Secret
+metadata:
+  labels:
+    liqo.io/remote-cluster-id: cool-firefly
+  name: kubeconfig-resourceslice-mypool
+  namespace: liqo-tenant-cool-firefly
+type: Opaque
+```
+
+### Delete VirtualNode
+
+You can revert the process by deleting the `VirtualNode` in the consumer cluster.
+
+```bash
+kubectl delete virtualnode mynode -n liqo-tenant-cool-firefly
+```
+
+## Multiple VirtualNodes
+
+You can create multiple `VirtualNodes` associated with the same provider cluster.
```

Comment:
In each of the section I would add a brief example explain when that case might be useful. For example, in this case, why should I create multiple virtual node?